### PR TITLE
DO-430: List issue fixed.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/BusinessList/BusinessListMobile/BusinessListCard/CardDetails/CardDetails.jsx [hunk 1]
```diff
@@ -157,8 +157,8 @@ const CardDetails = ({ cardDetails = {} }) => {
       <div className="business-card-detail__content">
         <BusinessName cardDetails={cardDetails} />
         <BusinessDescription description={description} />
-        <BusinessProducts products={products} />
-        <BusinessServices services={services} />
+        <span>{products ? <BusinessProducts products={products} /> : ''}</span>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/BusinessList/BusinessListMobile/BusinessListCard/CardDetails/CardDetails.jsx:160]: wrap it under BusinessProducts component.

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/BusinessList/BusinessListMobile/BusinessListCard/CardDetails/CardDetails.jsx [hunk 2]
```diff
@@ -157,8 +157,8 @@ const CardDetails = ({ cardDetails = {} }) => {
       <div className="business-card-detail__content">
         <BusinessName cardDetails={cardDetails} />
         <BusinessDescription description={description} />
-        <BusinessProducts products={products} />
-        <BusinessServices services={services} />
+        <span>{products ? <BusinessProducts products={products} /> : ''}</span>
+        <span>{services ? <BusinessServices services={services} /> : ''}</span>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/BusinessList/BusinessListMobile/BusinessListCard/CardDetails/CardDetails.jsx:161]: same here

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/_MemberListViewRoute.scss [hunk 3]
```diff
@@ -24,9 +24,9 @@
       transition: all .2s;
       background: #ffff;
       background-clip: border-box;
-      border: 1px solid rgba(26,54,126,0.125);
+      //border: 1px solid rgba(26,54,126,0.125);
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/_MemberListViewRoute.scss:27]: commented out code?

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/BusinessList/BusinessListMobile/BusinessListCard/CardDetails/CardDetails.jsx (hunk 1 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/BusinessList/BusinessListMobile/BusinessListCard/CardDetails/CardDetails.jsx]: done

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/BusinessList/BusinessListMobile/BusinessListCard/CardDetails/CardDetails.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/BusinessList/BusinessListMobile/BusinessListCard/CardDetails/CardDetails.jsx]: done

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/_MemberListViewRoute.scss (hunk 3 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/_MemberListViewRoute.scss]: done

**Pull request merged**